module-apps/apps-common/notifications/NotificationProvider.cpp

```diff
@@ -120,6 +120,25 @@ void NotificationProvider::handleSnooze(unsigned snoozeCount)
     send();
 }
 
+void NotificationProvider::handleBatteryTooHot(bool tooHot, bool allowUpdate)
```

Comment:
I suggest change name `tooHot` to `isBatteryTooHot`
`tooHot` could mean that some other temperature in device is too high and we assume that battery is hot too.
`is<sth>` is more clear name convention when using bool.

Reply:
Fixed!